packages/analytics-js-plugins/src/errorReporting/utils.ts

```diff
@@ -144,6 +144,10 @@ const getBugsnagErrorEvent = (
 
 const isRudderSDKError = (event: any) => {
   const errorOrigin = event.stacktrace?.[0]?.file;
+  const errorMessage = event.message;
+  if (errorMessage && typeof errorMessage === 'string' && errorMessage.includes('a non-error')) {
```

Comment:
The string "a non-error" doesn't look reliable enough. Do we have anything else?

Reply:
Why can't we use the error message filtering in the core SDK that we already have? We shouldn't filter errors this way. If needed, use regex based string matching.

Reply:
Addressed